diff --git a/src/plugins/plugins.pro b/src/plugins/plugins.pro
index 2095628..ea94668 100644
--- a/src/plugins/plugins.pro
+++ b/src/plugins/plugins.pro
@@ -28,6 +28,7 @@ SUBDIRS   = plugin_coreplugin \
             plugin_cpaster \
             plugin_cmakeprojectmanager \
             plugin_fakevim \
+	    plugin_emacskeys \
             plugin_designer \
             plugin_resourceeditor \
             plugin_genericprojectmanager \
@@ -167,6 +168,11 @@ plugin_fakevim.subdir = fakevim
 plugin_fakevim.depends = plugin_coreplugin
 plugin_fakevim.depends += plugin_texteditor
 
+plugin_emacskeys.subdir = emacskeys
+plugin_emacskeys.depends = plugin_projectexplorer
+plugin_emacskeys.depends += plugin_coreplugin
+plugin_emacskeys.depends += plugin_texteditor
+
 plugin_qtestlib.subdir = qtestlib
 plugin_qtestlib.depends = plugin_projectexplorer
 plugin_qtestlib.depends += plugin_coreplugin
